membersrvc/ca/ca_test.yaml

```diff
@@ -339,3 +339,8 @@ pki:
                             file: testdata/server1.key
 
             devops-address: 0.0.0.0:30303
+
+        ca:
+            subject:
+                organization: Hyperledger
+                country: US
```

Comment:
github seems to think that there's a missing newline to terminate the file. yet, I ran linter-js-yaml and it reports no errors on either file. am not a yaml expert, maybe someone else knows whether github's own linting is correct here or not. @srderson @binhn @ghaskins